library/Icinga/Web/StyleSheet.php

```diff
@@ -137,17 +144,20 @@ protected function collect()
             }
         }
 
-        if ($theme && $theme !== self::DEFAULT_THEME) {
-            if (($pos = strpos($theme, '/')) !== false) {
-                $moduleName = substr($theme, 0, $pos);
-                $theme = substr($theme, $pos + 1);
-                if ($mm->hasLoaded($moduleName)) {
-                    $module = $mm->getModule($moduleName);
-                    $this->lessCompiler->setTheme($module->getCssDir() . '/themes/' . $theme . '.less');
-                }
-            } else {
-                $this->lessCompiler->setTheme($this->pubPath . '/css/themes/' . $theme . '.less');
+        if ($themePath = self::getThemeFile($theme)) {
+                $this->lessCompiler->setTheme($themePath);
+        }
+
+        if ($user = Auth::getInstance()->getUser()) {
+            $mode = $user->getPreferences()->getValue('icingaweb', 'mode', 'none');
```

Comment:
Can't you just use the default mode as default value here? `none` shouldn't be possible to configure anyway.